### PR TITLE
Fix spelling in comments

### PR DESCRIPTION
**Pull request by Olf0**
mantainer -> maintainer

### CODE REVIEW

File: src/api/test/functional/search_controller_test.rb [hunk 1]
```diff
@@ -720,7 +720,8 @@ def test_search_for_binary_without_definition_yet
   def test_search_for_missing_role_defintions_in_all_visible_packages
     login_Iggy
 
-    # search for not mantainer packages
+    # search for not mai
```
Comment (dmarcoux) [inline src/api/test/functional/search_controller_test.rb:723]: This comment should be on 1 line.

Comment (Olf0) [reply src/api/test/functional/search_controller_test.rb]: Ooops, sure: Fixed the broken fix now.

**Review by danidoni — APPROVED** [review]

**Review by danidoni — CHANGES_REQUESTED** [review]
@Olf0, can you please squash all the commits into just one? Then it's good to go.

**Comment by Olf0** [comment]
> @Olf0, can you please squash all the commits into just one? Then it's good to go.

I cannot see how this can be achieved at Github's web-frontend.
As I do not have a local copy, I will leave it here.  Sorry.

Proceed with is as you like.

**Comment by krauselukas** [comment]
@Olf0 @danidoni squashed the commit for you

**Review by krauselukas — APPROVED** [review]

File: src/api/test/functional/search_controller_test.rb [hunk 2]
```diff
@@ -635,7 +635,7 @@ def test_find_owner
     assert_xml_tag tag: 'person', attributes: { name: 'king', role: 'maintainer' }
     assert_xml_tag tag: 'group', attributes: { name: 'test_group', role: 'maintainer' }
 
-    # search for not mantainer packages
+    # search for not maintainer packages
```
Comment (hennevogel) [inline src/api/test/functional/search_controller_test.rb:638]: While fixing the typo let's also fix the meaning. 
```suggestion
    # search for un-maintained packages
```

File: src/api/test/functional/search_controller_test.rb [hunk 3]
```diff
@@ -720,7 +720,7 @@ def test_search_for_binary_without_definition_yet
   def test_search_for_missing_role_defintions_in_all_visible_packages
     login_Iggy
 
-    # search for not mantainer packages
+    # search for not maintainer packages
```
Comment (hennevogel) [inline src/api/test/functional/search_controller_test.rb:723]:
```suggestion
    # search for un-maintained packages
```

**Review by hennevogel — CHANGES_REQUESTED** [review]

**Review by hennevogel — APPROVED** [review]

**Review by Olf0 — COMMENTED** [review]
Looking good.

**Pull request merged**